pallets/handles/src/rpc/src/lib.rs

```diff
@@ -80,11 +85,23 @@ where
 
 	fn get_next_suffixes(
 		&self,
-		handle_input: PresumptiveSuffixesRequest,
+		base_handle: String,
+		count: Option<u16>,
 	) -> RpcResult<PresumptiveSuffixesResponse> {
 		let api = self.client.runtime_api();
 		let at = BlockId::hash(self.client.info().best_hash);
-		let suffixes_result = api.get_next_suffixes(&at, handle_input);
+		let handle_input: Handle = base_handle.into_bytes().try_into().unwrap();
+		let max_count = MAX_SUFFIXES_COUNT;
+		let count = count.unwrap_or(max_count).min(max_count);
```

Comment:
Take lowest of two

Reply:
Hmm... Let's not give them more than they asked for. `0` doesn't make sense, but if our default is `1` even if we increase it in the future we don't have a breaking change.